src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/LinkFactoryImpl.java

```diff
@@ -112,7 +112,7 @@ protected Content getClassLink(LinkInfo linkInfo) {
                         link.add(m_writer.links.createLink(
                                 filename.fragment(classLinkInfo.where),
                                 label,
-                                classLinkInfo.isStrong,
+                                classLinkInfo.style,
```

Comment:
I guess we're lucky the code is downcasting the `LinkInfo` into `LinkInfoImpl`.

It would be a separate cleanup/rename, but this might be another place where the subtype might be better named `HtmlLinkInfo` rather than just `LinkInfoImpl` and this class might be `HtmlLinkFactory`